src/mmw/apps/geoprocessing_api/calcs.py

```diff
@@ -86,6 +86,7 @@ def stream_data(geojson):
         str(s['stream_order']): {
             "order": s['stream_order'],
             "lengthkm": s['lengthkm'],
+            "ag_stream_pct": results['ag_stream_pct'],
```

Comment:
We should also add `ag_stream_pct` to the example completed job JSON in the Swagger docs: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/geoprocessing_api/views.py#L605

Reply:
Yes we should! You are 💯

Reply:
Good catch, added in 6fdf4eb. Will merge shortly. Thanks for reviewing!